src/main/java/innohack/gem/entity/extractor/ExtractConfig.java

```diff
@@ -10,10 +10,12 @@
   private List<TimestampColumn> timestampColumns;
   private int groupId;
   private Extractor extractor;
+  private List<String> sheetNames;
```

Comment:
sheetname should be taken from parameters of AbstractExtractor instead of a attribute in ExtractConfig. need to merge #178 to see Parameters in AbstractExtractor

Reply:
Rem to remove sheetname attribute from extractconfig

Reply:
ok done